### PR TITLE
[WIP] CWP chapters: clarify the how-to

### PR DESCRIPTION
**Pull request by jouvin**
- In particular, explain how to generate the author list with a2tex

### CODE REVIEW

File: cwp/cwp-wg-papers.md [hunk 1]
```diff
@@ -22,16 +22,52 @@ recommended.
...
+   * In the previous line, `Affiliation_ID` refers to the first word in the
+   [affiliation list](https://github.com/HSF/documents/tree/master/CWP/papers/HSF-CWP-2017-01_roadmap/authors/addresses.txt) 
+   and `footnote_key` to the number at the start of the line in the
+   [footnote list](https://github.com/HSF/documents/tree/master/CWP/papers/HSF-CWP-2017-01_roadmap/authors/footnotes.txt). 
+   If there are several affiliations, use ` & ` as the separator. If there are several 
+   footnotes, separate them with a comma without surrounding spaces.
+
+1. To create the title page use the `\maketitle` statement (without arguments). **Before** 
+you need to define the title, the abstract and the author list:
+  * The title must be `HEP Software Foundation Community White Paper Working Group \-- *My Working Group*`.
```
Comment (graeme-a-stewart) [inline cwp/cwp-wg-papers.md]: I just noticed this is a little weird. Should remove the backslash and the asterisks.

File: cwp/cwp-wg-papers.md [hunk 2]
```diff
@@ -22,16 +22,52 @@ recommended.
...
       odd non-breaking spaces appear. Just delete these.
-1. Adopt the JHEP preprint style, using the template in the
+1. Adopt the [JHEP preprint](https://jhep.sissa.it/jhep/help/JHEP/TeXclass/DOCS/JHEP-author-manual.pdf)
+   style, using the template in the
    [software development paper](https://github.com/HSF/documents/tree/master/CWP/papers/HSF-CWP-2017-13_soft-dev/latex/).
     - Copy the `jheppub.sty` file into your latex area as it is needed for arXiv side compilation.
-1. The paper title should be `HEP Software Foundation Community White Paper Working Group \-- *My Working Group*`.
-1. You will need to add an abstract for arXiv.
-1. Add a copyright and license statement as a comment - we strongly recommend <br>`% Copyright (C) 2018, The HSF Community White Paper authors, licence CC-BY-4.0.`
+1. Build the author list: the **strongly recommended** way is to create a text file
```
Comment (graeme-a-stewart) [inline cwp/cwp-wg-papers.md]: Give an example here - the DOMA paper would be ideal so that people have a clearer idea how this work,

Comment (jouvin) [reply cwp/cwp-wg-papers.md]: Added

**Comment by jouvin** [comment]
@graeme-a-stewart I missed that you merged it when it was still WIP... I'm pushing my last changes as a separate PR...

**Comment by graeme-a-stewart** [comment]
Damn it - sorry, I didn't notice. (It's a nice feature of GitLab that WIPs cannot be merged at all.)

**Pull request merged**